tests/unit/nupic/research/inhibition_object_test.py

```diff
@@ -71,26 +73,25 @@ def testInhibition(self):
     spVerbosity = 0
     testIter = 100
 
-    spTest = FDRCSpatial2.FDRCSpatial2(
-                                coincidencesShape=(coincidencesShape, 1),
-                                inputShape = (1, inputLen),
-                                inputBorder = inputLen/2 - 1,
-                                coincInputRadius = inputLen / 2,
-                                numActivePerInhArea = numActivePerInhArea,
-                                spVerbosity = spVerbosity,
-                                stimulusThreshold = stimulusThreshold,
-                                seed = spSeed
-                                )
+    spTest = SpatialPooler(
+                           columnDimensions=(columnDimensions, 1),
+                           inputDimensions = (1, inputLen),
```

Comment:
No spaces around `=` for keyword args here and elsewhere (I realize this is a remnant from old code but might as well fix while you're touching these lines).